packages/cadl-ranch-specs/http/projection/projected-name/main.tsp

```diff
@@ -128,3 +128,51 @@ op parameter(
   @query
   `default-name`: string
 ): NoContentResponse;
+
+@route("/model")
+@operationGroup
+namespace Model {
+  @projectedName("client", "_DontUseMeAsAName_") // Should be ignored
+  @projectedName("csharp", "CSModel")
+  @projectedName("java", "JavaModel")
+  @projectedName("javascript", "TSModel")
+  @projectedName("python", "PythonModel")
```

Comment:
should we add projectedName json for it. though it might not impact any of the dpg codegen, as model name doesn't show up in payload.

Reply:
Hello @qiaozha It will affect the model name. Codegen will generate the model with the name specified in `projectedName`. e.g csharp will generate a model named `CSModel` instead of `ModelWithLanguageProjectedName`. It will not affect the payload, but it affects the code generated and the signature in the convenience method.

Reply:
Interesting, Are you suggesting csharp will respect projectedName json. I thought it will just ignore json name ?